stdlib/2and3/colorsys.pyi

```diff
@@ -0,0 +1,17 @@
+# Stubs for colorsys
+
+from typing import Tuple, Union
+
+def yiq_to_rgb(y: float, i: float, q: float) -> Tuple[float, float, float]: ...
+def rgb_to_hls(r: float, g: float, b: float) -> Tuple[float, float, float]: ...
+def hls_to_rgb(h: float, l: float, s: float) -> Tuple[float, float, float]: ...
+def rgb_to_hsv(r: float, g: float, b: float) -> Tuple[float, float, float]: ...
+def hsv_to_rgb(h: float, s: float, v: float) -> Tuple[float, float, float]: ...
+
+# TODO undocumented
+ONE_SIXTH = ...  # type: float
+ONE_THIRD = ...  # type: float
+TWO_THIRD = ...  # type: float
+
+# TODO undocumented, should only work with float
+def rgb_to_yiq(r: complex, g: complex, b: complex) -> Tuple[float, float, float]: ...
```

Comment:
Actually, the result is complex, too, if the input is complex.

It's probably not worth it to model that with an `@overload` or `TypeVar`. I'm fine with it if you just change it back to float. Nobody is *supposed* to call this function with a complex, anyway. Sorry for the back and fro.